sdk/python/kfp/compiler/compiler.py

```diff
@@ -608,7 +608,7 @@ def _create_volumes(self, pipeline):
     volumes.sort(key=lambda x: x['name'])
     return volumes
 
-  def _create_pipeline_workflow(self, args, pipeline, op_transformers=None):
+  def _create_pipeline_workflow(self, args, pipeline, op_transformers=None, pipeline_conf=None):
```

Comment:
What will happen if a user set op transformer in op_transformers and also in pipeline_conf?

Reply:
This api is hidden, but is it possible for a user to call dsl.get_pipeline_conf().add_op_transformer(gcp.use_gcp_secret('user-gcp-sa')) inside the pipline definition, and then do something not compatible in pipeline_conf?

Reply:
Yes this is possible.
In this case the configuration explicitly specified by the "pipeline run submitter" overrides possible configuration set by the "pipeline author".

Reply:
Added docstring describing the behavior.